src/lamplib/src/genny/tasks/run_tests.py

```diff
@@ -201,80 +208,43 @@ def _setup_resmoke(
             check=True,
             capture=False,
         )
-        cmd_runner.run_command(
-            # If changing this sha, you may need to use later binaries
-            # in the _canned_artifacts dict.
-            cmd=["git", "checkout", "cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3"],
-            cwd=mongo_repo_path,
-            check=True,
-            capture=False,
-        )
-    else:
-        SLOG.info("Using existing mongo repo checkout", mongo_repo_path=mongo_repo_path)
-        cmd_runner.run_command(
-            cmd=["git", "rev-parse", "HEAD"], check=False, cwd=mongo_repo_path, capture=False,
-        )
-
-    # Look for mongod in
-    # build/opt/mongo/db/mongod
-    # build/install/bin/mongod
-    # bin/
-    opt = os.path.join(mongo_repo_path, "build", "opt", "mongo", "db", "mongod")
-    install = os.path.join(mongo_repo_path, "build", "install", "bin", "mongod")
+    cmd_runner.run_command(
```

Comment:
Is this unconditionally checking out the mongo repo? I think if someone passing `--mongo-repo-path` that means they have an existing mongo repo they want to use, possibly on a specific commit, and we shouldn't do a checkout?

We could probably eliminate the mongo commit option and just always use the pinned commit or whatever the user has checked out. I'm wary about running git inside someone's repo and maybe messing things up